x-pack/plugins/security/public/management/api_keys/api_keys_api_client.ts

```diff
@@ -10,6 +10,7 @@ import { ApiKey, ApiKeyToInvalidate } from '../../../common/model';
 interface CheckPrivilegesResponse {
   areApiKeysEnabled: boolean;
   isAdmin: boolean;
+  canManage: boolean;
```

Comment:
**note** previously, the API Keys app had a dual purpose for `areApiKeysEnabled`: It was true if and only if both of the following were true:
- User was authorized to retrieve their own API Keys
- API Keys were in fact enabled

This separates those two concerns into dedicated properties